lib/membrane_rtc_engine/endpoints/webrtc_endpoint.ex

```diff
@@ -47,6 +47,10 @@ defmodule Membrane.RTC.Engine.Endpoint.WebRTC do
   * [Typescript library intended for use in web browsers](https://github.com/membraneframework/membrane-webrtc-js)
   * [Android native library](https://github.com/membraneframework/membrane-webrtc-android)
   * [IOS native library](https://github.com/membraneframework/membrane-webrtc-ios)
+
+  ## Monitoring track activity
+  WebRTC Endpoint only monitors simulcast tracks activity, meaning that it never emits `Membrane.RTC.Engine.Event.TrackVariantPaused` event
+  for non-simulcast tracks. The main reason is that it's very hard to distinguish inactive screen sharing from screen sharing of static content.
```

Comment:
```suggestion
  for non-simulcast tracks. The main reason is that it's impossible to tell if the screensharing track is inactive or not sending any packets because it contains static content.
```